plugin/Model/FormModelBuilder/FormModelBuilder.php

```diff
@@ -125,16 +125,29 @@ protected function readTemplatePaths($directory, $pattern = '*')
 		$templatesAll = array_merge($templateGlobFiles, $templateLocalFiles, $templateThemeFiles);
 		$templates = array();
 
+		$plugin_name = basename(plugin_dir_path(ONOFFICE_PLUGIN_DIR . '/index.php'));
 		foreach ($templatesAll as $value) {
+			$file = substr(strrchr($value, "/"), 1);
 			if(strpos($value, 'themes') !== false) {
 				$value = __String::getNew($value)->replace(get_template_directory().'/', '');
-				$templates[$value] = $value;
```

Comment:
Does it work to hardcode `$template[1]`, `$template[2]` and `$template[3]` here?

Reply:
Keys 1, 2, 3 are the order of the template folders when displayed. Maybe I'll try using an array to store order of the template folders instead of hardcode